avocado/core/loader.py

```diff
@@ -207,16 +223,10 @@ def get_base_keywords(self):
         return base_path
 
     def get_type_label_mapping(self):
-        mapping = {}
-        for loader_plugin in self._initialized_plugins:
-            mapping.update(loader_plugin.get_type_label_mapping())
-        return mapping
```

Comment:
The mapping depends only on `_initialized_plugins` and is used several times. It makes no sense to evaluate it every time we call the `get_*` function so I believe it's better to do this when `_initialized_plugins` is updated.

Basically it's a question whether we want to initialize this lazily, or right away. I believe the right away is better in this case as we know for sure they are static.